### ISSUE TITLE
How to set up obfs-uri ? any examples?

### ISSUE THREAD

**Issue by manjuprajna** [shadowsocks/simple-obfs#168]
server side remains unchanged, 
client side:
"plugin_opts":"obfs=http;obfs-host=www.bing.com;obfs-uri=*******;fast-open"
like this way?
but is there any "official" examples? what should I put after "obfs-uri="?

thanks!

**Comment by slurin** [comment]
The server do not check the request path at all, thus, no matter what the request path is, the server can handle the connection as well.
you can simply add `obfs-uri` on client-side without any change on the server.
If your plugin_opts are: `obfs=http;obfs-host=www.bing.com;obfs-uri=/path/to/file;`, the first line of the client request header should be `GET /path/to/file HTTP/1.1`.

**Comment by manjuprajna** [comment]
@slurin I understand, so server does not care what this uri parameter is, because it's used to cheat ISP's facilities to make shadowsocks data packets look like a normal http packets right? so the server side will strip this fake http header out and get the real data
so I should simply modify the config.json of client side like this:
**"plugin_opts":"obfs=http;obfs-host=www.bing.com;obfs-uri=/default.aspx;fast-open"**
anything like that right?
so why this commit add uri function is to make obfs data packets look more real, right?

**Comment by slurin** [comment]
Yeah, hardly any servers use path uri `/` as a websocket's entry point, especially on HTTP server port 80.

**Comment by debiansid** [comment]
after setup uri , can I setup http(s) proxy redirect to different backend?

**Comment by slurin** [comment]
There's no change in the server side, you can use `failover` as usual.

**Comment by debiansid** [comment]
@slurin 
obfs-local will send uri to server? then I would test it for lighttpd on server side.

**Comment by corerise** [comment]
what did u put in the uri section?
